Code/Framework/AzToolsFramework/AzToolsFramework/UI/Outliner/EntityOutlinerWidget.cpp

```diff
@@ -1140,9 +1148,9 @@ namespace AzToolsFramework
 
     void EntityOutlinerWidget::OnPrefabInstancePropagationEnd()
     {
-        QTimer::singleShot(1, this, [this]() {
+        QTimer::singleShot(10, this, [this]() {
             m_gui->m_objectTree->setUpdatesEnabled(true);
-            m_gui->m_objectTree->expand(m_proxyModel->index(0,0));
+            m_gui->m_objectTree->expandToDepth(0);
```

Comment:
question: How come this had to move to 10? Is it worth adding a comment for this? Does it break if it's 1? 😛